### ISSUE TITLE
HTTP 410 Client Error: Gone after GitLab V3 API deprecation

### ISSUE THREAD

**Issue by danirod** [redmine-gitlab-migrator/redmine-gitlab-migrator#31]
[GitLab API v3 was deprecated on GitLab 9.0](https://docs.gitlab.com/ee/api/v3_to_v4.html) and it's being removed on GitLab 11.0.

Running migrate-rg to migrate milestones and issues from Redmine to a GitLab instance running GitLab 11.0 (such as GitLab.com, which is running GitLab 11.0.0-rc12-ee at the moment), causes an exception to be raised:

```
Traceback (most recent call last):
  File "/home/danirod/.local/share/virtualenvs/p3-uq_ooE2q/bin/migrate-rg", line 11, in <module>
    sys.exit(main())
  File "/home/danirod/.local/share/virtualenvs/p3-uq_ooE2q/lib/python3.5/site-packages/redmine_gitlab_migrator/commands.py", line 237, in main
    args.func(args)
  File "/home/danirod/.local/share/virtualenvs/p3-uq_ooE2q/lib/python3.5/site-packages/redmine_gitlab_migrator/commands.py", line 212, in perform_migrate_roadmap
    gitlab_project=gitlab_project)
  File "/home/danirod/.local/share/virtualenvs/p3-uq_ooE2q/lib/python3.5/site-packages/redmine_gitlab_migrator/commands.py", line 72, in check
    ret = func(redmine_project, gitlab_project)
  File "/home/danirod/.local/share/virtualenvs/p3-uq_ooE2q/lib/python3.5/site-packages/redmine_gitlab_migrator/commands.py", line 94, in check_no_milestone
    return len(gitlab_project.get_milestones()) == 0
  File "/home/danirod/.local/share/virtualenvs/p3-uq_ooE2q/lib/python3.5/site-packages/redmine_gitlab_migrator/gitlab.py", line 117, in get_milestones
    '{}/milestones'.format(self.api_url))
  File "/home/danirod/.local/share/virtualenvs/p3-uq_ooE2q/lib/python3.5/site-packages/redmine_gitlab_migrator/gitlab.py", line 16, in get
    return super().get(*args, **kwargs)
  File "/home/danirod/.local/share/virtualenvs/p3-uq_ooE2q/lib/python3.5/site-packages/redmine_gitlab_migrator/__init__.py", line 37, in get
    return self._req(requests.get, *args, **kwargs)
  File "/home/danirod/.local/share/virtualenvs/p3-uq_ooE2q/lib/python3.5/site-packages/redmine_gitlab_migrator/__init__.py", line 31, in _req
    resp.raise_for_status()
  File "/home/danirod/.local/share/virtualenvs/p3-uq_ooE2q/lib/python3.5/site-packages/requests/models.py", line 939, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 410 Client Error: Gone for url: https://gitlab.com/api/v3/projects/[redacted]?per_page=100
```

**Comment by romanr** [comment]
Any change of merging it so we can try using the migrator? the whole project is unusable right now.

**Comment by awsom82** [comment]
same here

**Comment by yossizap** [comment]
This issue was recently fixed and merged in #35
